spyderlib/qt/__init__.py

```diff
@@ -9,7 +9,7 @@
 
 import os
 
-os.environ.setdefault('QT_API', 'pyqt')
+os.environ.setdefault('QT_API', 'pyqt5')
```

Comment:
As @ccordoba12 said, our support for pyqt5 is experimental, meaning there are many bugs that need to be fixed before we make it a default, so there is no pressing need to have this merged if the distributions that support it cannot (yet) provide packages.